libcontainer/cgroups/fs2/cpu.go

```diff
@@ -91,3 +91,11 @@ func statCpu(dirPath string, stats *cgroups.Stats) error {
 	}
 	return nil
 }
+
+func statCpuPressure(dirPath string, stats *cgroups.Stats) error {
+	const file = "cpu.pressure"
```

Comment:
I would pass "cpu.pressure" directly, without adding an intermediary `const`. The `const` would make sense if we use it more than once, or if the value is complicated/long -- neither is the case here.